debian/raspberrypi-net-mods.service

```diff
@@ -1,13 +1,13 @@
 [Unit]
 Description=Copy user wpa_supplicant.conf
-ConditionPathExists=/boot/wpa_supplicant.conf
+ConditionPathExistsGlob=/boot/wpa_supplicant.conf{,.txt}
 Before=dhcpcd.service
 After=systemd-rfkill.service
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/bin/mv /boot/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.conf
+ExecStart=/usr/lib/raspberrypi-net-mods/rpnm.sh
```

Comment:
A more meaningful filename (e.g. `cp_wpa.sh`) might be useful?

Reply:
Renamed to mv-wpa.sh, since it's a move.